ci/jenkins/README.md

```diff
@@ -105,7 +106,8 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-conformance --registry ${DOCKER_REGISTRY}
 ```
 
-* windows network policy: community tests on Windows cluster with Docker runtime, focusing on "Feature:NetworkPolicy".
+* windows network policy: community tests on Windows cluster with Containerd runtime,
+  and OVS running inside container focusing on "Feature:NetworkPolicy".
```

Comment:
```suggestion
* Windows network policy: community tests on Windows cluster with containerd runtime,
  and OVS running inside a container focusing on "Feature:NetworkPolicy".
```
make the `containerd` consistent. I think the product doesn't start with a upper case.